### ISSUE TITLE
r5s not supported

### ISSUE THREAD

**Issue by jvivian** [DataBiosphere/toil#2429]
Using auto-scaling to do some preprocessing and ran into issues using r5/r5ds.  R4 instances work just fine.

```
INFO:toil.lib.ec2:Creating r5d.4xlarge instance(s) ...
ERROR:toil.provisioners.clusterScaler:Exception encountered in scaler thread. Making a best-effort attempt to keep going, but things may go wrong from now on.
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/toil/provisioners/clusterScaler.py", line 877, in tryRun
    self.scaler.updateClusterSize(estimatedNodeCounts)
  File "/usr/local/lib/python2.7/dist-packages/toil/provisioners/clusterScaler.py", line 546, in updateClusterSize
    newNodeCount = self.setNodeCount(nodeType=nodeType, numNodes=estimatedNodeCount, preemptable=nodeShape.preemptable)
  File "/usr/local/lib/python2.7/dist-packages/toil/provisioners/clusterScaler.py", line 616, in setNodeCount
    preemptable=preemptable)
  File "/usr/local/lib/python2.7/dist-packages/toil/provisioners/clusterScaler.py", line 632, in _addNodes
    return self.provisioner.addNodes(nodeType=nodeType, numNodes=numNodes, preemptable=preemptable)
  File "/usr/local/lib/python2.7/dist-packages/toil/provisioners/aws/awsProvisioner.py", line 274, in addNodes
    spec=kwargs, num_instances=numNodes)
  File "/usr/local/lib/python2.7/dist-packages/toil/lib/ec2.py", line 365, in create_ondemand_instances
    **spec).instances
  File "/usr/local/lib/python2.7/dist-packages/boto/ec2/connection.py", line 977, in run_instances
    verb='POST')
  File "/usr/local/lib/python2.7/dist-packages/boto/connection.py", line 1208, in get_object
    raise self.ResponseError(response.status, response.reason, body)
EC2ResponseError: EC2ResponseError: 400 Bad Request
<?xml version="1.0" encoding="UTF-8"?>
<Response><Errors><Error><Code>UnsupportedOperation</Code><Message>The instance configuration for this AWS Marketplace product is not supported. Please see the AWS Marketplace site for more information about supported instance types, regions, and operating systems.</Message></Error></Errors><RequestID>bbd90aad-edf4-4267-9c96-0106b7d88c46</RequestID></Response>
```

┆Issue is synchronized with this [JIRA Story](https://ucsc-cgl.atlassian.net/browse/TOIL-380)
┆Issue Number: TOIL-380

**Comment by DailyDreaming** [comment]
@jvivian Says it exists: https://github.com/DataBiosphere/toil/blob/96d817b639acd11a1b97b07b0adfe7eec9706cf2/src/toil/lib/generatedEC2Lists.py#L132

Are you using the latest version of toil?

**Comment by jvivian** [comment]
@DailyDreaming — I'm using `3.18.0`, but didn't think about region possibilities. I'm in us-west-2 which [this page](https://aws.amazon.com/about-aws/whats-new/2018/09/amazon-ec2-r5-and-r5d-instances-now-available-in-8-additional-aws-regions/) says should have r5 instances.

**Comment by joelarmstrong** [comment]
I did some trial and error, and I think the issue is the AMI that toil's using by default might be a bit too old to support the r5 series. If you use `TOIL_AWS_AMI=ami-05507591f0fcb2b75` it should work.

**Comment by DailyDreaming** [comment]
@joelarmstrong Thanks for the sleuthing!  I'll make a PR updating the default AMI.

**Comment by DailyDreaming** [comment]
Should now be fixed by #2430.  Closing this.
